gateway/bin/main.rs

```diff
@@ -22,7 +22,7 @@ extern crate fdlimit;
 extern crate signal_hook;
 #[macro_use]
 extern crate clap;
-extern crate failure;
+extern crate anyhow;
```

Comment:
```suggestion
```

Reply:
why? [gateway](https://github.com/oasislabs/oasis-ethwasi-runtime/blob/master/gateway/Cargo.toml) is not on 2018 edition (not sure why though).